sources/packages/api/src/route-controllers/confirmation-of-enrollment/confirmation-of-enrollment.controller.ts

```diff
@@ -215,13 +219,19 @@ export class ConfirmationOfEnrollmentController {
       );
     }
 
-    await this.applicationService.updateCOEStatus(
-      applicationId,
-      COEStatus.submitted,
-    );
+    const updatedCOEStatus =
+      await this.applicationService.updateApplicationCOEStatus(
+        applicationId,
+        COEStatus.completed,
+        ApplicationStatus.completed,
+      );
 
-    // Send a message to allow the workflow to proceed.
-    await this.workflow.sendConfirmCOEMessage(application.assessmentWorkflowId);
+    if (updatedCOEStatus) {
```

Comment:
This condition will be always true since _this.applicationService.updateApplicationCOEStatus_ will always return an object with the status of the update.